app/src/main/java/com/github/se/gomeet/ui/mainscreens/events/MyEventInfo.kt

```diff
@@ -266,6 +292,37 @@ fun MyEventInfo(
   if (showShareEventDialog) {
     ShareDialog("Event", eventId, onDismiss = { showShareEventDialog = false })
   }
+
+  if (showDeleteEventDialog) {
+    DeleteEventDialog(
+        onConfirm = {
+          coroutineScope.launch {
+            myEvent.value!!.participants.forEach { participant ->
```

Comment:
What happens in the onConfirm lambda should really be a function in EventViewModel, but technically that's a convention more than anything else, so as long as it works it's all good and we can refactor it later.